database/init_db.go

```diff
@@ -50,7 +50,7 @@ func main() {
 
 	// SQL instruction to create a table.
 	sqlStatement := `
-	CREATE TABLE IF NOT EXISTS public.test_objects (
+	CREATE TABLE IF NOT EXISTS resource (
```

Comment:
I'm not sure about what happens here if we create a table outside the `public` schema.

Reply:
I'll change it to public.resource